docker/Dockerfile.base

```diff
@@ -2,7 +2,7 @@
 # and scripts/setup_macos.sh, if there's any changes applied to this file,
 # specially regarding the installation of dependencies, apply those same
 # changes to the mentioned files.
-ARG PARENT_IMAGE=ubuntu:16.04
+ARG PARENT_IMAGE=ubuntu:18.04
```

Comment:
can you branch these into Dockerfile.base.18.04, Dockerfile.base.16.04, etc? we will want to keep both configs around for a while. we can have the Makefile and CI use 18.04 by default.

Reply:
Yes, that was the plan. This was just a quick push to check if things were working while i fix my local environment.